simple_budget.rb

```diff
@@ -46,7 +46,11 @@ def combined_monthly_func(number_of_people, members)
       # "monthly_net_income" element of each user's grouping in the array "members"
       combined_monthly += members[user][1]
     end
+  elsif 

```

Comment:
This was the main change. Adding the elsif allows for a case where a single user is trying to generate a budget.

Reply:
Thank you, I'll definitely add this change